CODE_OF_CONDUCT.md

```diff
@@ -35,8 +42,165 @@ Sage can more effectively collaborate with others if they follow this code.
+
+Standards for behavior in the Sage community are detailed in the Code
+of Conduct above. Participants in our community should uphold these
+standards in all their interactions and help others to do so as well.
+
+## Reporting guidelines ##
+
+We know that it is painfully common for internet communication to
+start at or devolve into obvious and flagrant abuse. We also recognize
+that sometimes people may have a bad day, or be unaware of some of the
+guidelines in this Code of Conduct. Please keep this in mind when
+deciding on how to respond to a breach of this Code.
+
+For clearly intentional breaches, report those to the Sage Code of
+Conduct Committee (see below). For possibly unintentional breaches,
+you may reply to the person and point out this Code of Conduct (either
+in public or in private, whatever is most appropriate). If you would
+prefer not to do that, please feel free to report to the Sage Code of
+Conduct Committee directly, or ask the committee for advice, in
+confidence.
+
+You can report issues to the Sage Code of Conduct Committee at
+sage-conduct@googlegroups.com. Currently, the committee consists of:
+
+- Vincent Delecroix
+- David Joyner
+- John Palmieri
+- David Roe
+- William Stein
+
+If your report involves any members of the committee, or if they feel
+they have a conflict of interest in handling it, then they will recuse
+themselves from considering your report. Alternatively, if, for any
+reason, you feel uncomfortable making a report to the whole committee,
+then you can also contact individual committee members.
+
+## Incident reporting resolution & Code of Conduct enforcement ##
+
+This section summarizes the most important points; more details can be
+found in the Code of Conduct enforcement guide.
+
+The Sage Code of Conduct Committee will investigate and respond to all
+complaints. The committee will protect the identity of the reporter,
+and treat the content of complaints as confidential (unless the
+reporter agrees otherwise).
+
+In case of severe and obvious breaches, e.g., personal threat or
+violent, sexist or racist language, the committee will immediately
+disconnect the originator from Sage communication channels; please see
+the manual for details.
+
+In cases not involving clear severe and obvious breaches of this Code
+of Conduct, the process for acting on any received Code of Conduct
+violation report will be:
+
+- acknowledgement that the report has been received
+- discussion within the committee
+- discussion with and/or feedback provided to the reportee
+- mediation (if feedback didn’t help, and only if both reporter and
+  reportee agree to this)
+- enforcement via transparent decision by the Sage Code of Conduct Committee
+
+The committee will respond to any report as soon as possible, and our
+goal will be to respond within 72 hours.
+
+Potential consequences for violating the Sage Code of Conduct include:
+
+- Nothing (if the committee determines that no violation occurred)
```

Comment:
This seems logically contradictory to the line above. Does the line above need to be "Potential consequences of alleged violations of the Sage Code of Conduct"?
Probably not, but then the "nothing" line, which is valuable, needs to be listed separately.

Reply:
I can add "alleged"; that's simpler than listing it separately.

Reply:
Actually, I'm going to replace "if the committee determines that no violation occurred" with "for example if the matter has been resolved publicly while the committee was considering responses" — this is in the second document already.